flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/test/java/com/ververica/cdc/connectors/values/source/ValuesDataSourceHelperTest.java

```diff
@@ -56,17 +56,14 @@ public void after() {
         ValuesDatabase.clear();
     }
 
-    @Test
-    public void testSingleSplitSingleTable() throws Exception {
+    /** read Events from {@link ValuesDataSource} and apply the events to ValuesDatabase. */
+    private void executeDataStreamJob(ValuesDataSourceHelper.SourceEventType type)
```

Comment:
Looks like these tests are ITCase instead of unit test, could we update the class name?